Packs/AzureStorageContainer/ReleaseNotes/1_0_20.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Azure Storage Container
+
```

Comment:
Note that you need to update about the docker update in the release notes as well.